hugo/layouts/partials/footer.html

```diff
@@ -8,14 +8,30 @@
         <div class="row no-gutters">
             <div class="col-sm">
                 <ul>
-                    <li><a class="footer-main" href="engine">source{d} Engine</a></li>
+                    <li><a class="footer-main" href="features">Product</a></li>
+                    <li><a href="features">Features</a></li>
+                    <li><a href="get-started">source{d} for Developers</a></li>
+                    <li><a href="get-started">source{d} for Managers</a></li>
+                    <li><a href="get-started">source{d} for Executives</a></li>
+                </ul>
+            </div>
+            <div class="col-sm pr-3">
+                <ul>
+                    <li><a class="footer-main" href="enterprise">Solutions</a></li>
+                    <li><a href="enterprise">source{d} Enterprise</a></li>
+                    <li><a href="enterprise#solutions">Application Portfolio Management</a></li>
```

Comment:
All four links inside `Enterprise` menu points to the same section.
Is it correct?